src/vid2805-dual-stepper-element.stories.ts

```diff
@@ -0,0 +1,88 @@
+import { html } from 'lit';
+import './vid2805-dual-stepper-element';
+import { StepperHand } from './vid2805-dual-stepper-element';
+
+export default {
+  title: 'VID2805 Dual Stepper',
+  component: 'wokwi-vid2805-dual-stepper',
+  argTypes: {
+    outerLength: { control: { type: 'range', min: 20, max: 70 } },
+    outerAngle: { control: { type: 'range', min: 0, max: 360 } },
+    outerColour: { control: { type: 'color' } },
+    outerShape: { options: ['arrow', 'plain', 'fancy'], control: { type: 'select' } },
+    innerLength: { control: { type: 'range', min: 20, max: 70 } },
+    innerAngle: { control: { type: 'range', min: 0, max: 360 } },
+    innerColour: { control: { type: 'color' } },
+    innerShape: { options: ['arrow', 'plain', 'fancy'], control: { type: 'select' } },
```

Comment:
Seems like all these controls are non functional - see my comment on the `outerHand` / `innerHand` properties in `vid2805-dual-stepper-element.ts` below.

Also, `fancy` is not implements for the shapes, so please remove it

Reply:
done - removed composite controls and now using only individual properties